### PR TITLE
Admin - Category - Create Category

### PR DESCRIPTION
**Pull request by at-vudang95**

### CODE REVIEW

File: app/Http/Requests/CategoryRequest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     */
+    public function messages()
+    {
+        return [
+            'name.required' => ' The category name field is required.',
+            'name.max' => ' The category name may not be greater than 255 characters.',
+            'name.unique' => ' The category name is existed.',
+        ];
+    }
+}
```
Comment (ghost) [inline app/Http/Requests/CategoryRequest.php]: What message error will be display when i using Vietnamese not English ❓

File: resources/views/categories/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                            <div class="form-group {{ $errors->has('description') ? 'has-error' : '' }}">
+                                <label for="description">{{__('Description')}}</label>
+                                <textarea class="form-control"
+                                          name="description">{{ old('description') }}</textarea>
+                                <span class="text-danger">{{ $errors->first('description') }}</span>
+                            </div>
+                        </div>
+                        <!-- /.box-body -->
+                        <div class="box-footer text-center">
+                            <input type="submit" class="btn btn-primary" value="Create">
```
Comment (ghost) [inline resources/views/categories/create.blade.php]: Should you use language for ```value="Create"```

File: app/Http/Requests/CategoryRequest.php (hunk 1 above)
Comment (at-vudang95) [reply app/Http/Requests/CategoryRequest.php]: Team think Admin page uses only English! How do you think?

File: resources/views/categories/create.blade.php (hunk 2 above)
Comment (at-vudang95) [reply resources/views/categories/create.blade.php]: Team think Admin page uses only English! How do you think?

**Comment by ghost** [comment]
I think we defined language in admin page. we need apply all label with language, not exception.

**Comment by at-vudang95** [comment]
I will edit. Thank you!

**Comment by at-vudang95** [comment]
With message validation, we will create and write in resources/lang/vi/validation.php.
With text other in view, we use string key and will create and write in resources/lang/vi.json.
They will pull request in another branch.

File: app/Http/Controllers/CategoryController.php [hunk 3]
```diff
@@ -6,5 +6,43 @@
...
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request Request from client
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CategoryRequest $request)
+    {
+        $this->category->create($request->all());
+        return redirect()->route('categories.index');
```
Comment (ghost) [inline app/Http/Controllers/CategoryController.php]: If can not create, what happen display ?

**Comment by ghost** [comment]
LGTM,
Thanks

**Pull request merged**